index.js

```diff
@@ -2,35 +2,50 @@
 //console.log(msg);
 var express = require('express');
 var bodyParser = require('body-parser');
+var DataStore = require('nedb');
 
 var port = 3000;
 var BASE_API_PATH = "/api/v1";
-
-var contacts = [
-    {"name" : "peter","phone" : "12456"},
-    {"name" : "jhon" , "phone": "6666"}
-];
+var DB_FILE_NAME =__dirname + "/contacts.json";
 
 console.log("Starting API server..");
 
 var app = express();
 app.use(bodyParser.json());
 
+var db = new DataStore({
+    filename: DB_FILE_NAME,
+    autoload: true
+});
+
 app.get("/", (req,res) =>{
     res.send("<html><body><h1>My server</h1></body></html>");
 });
 
 app.get(BASE_API_PATH + "/contacts",(req,res)=>{
     console.log(Date() + " - GET /contacts");
-    res.send(contacts);
+    db.find({} , (err,contact)=>{
+        if(err){ 
+            console.log(Date() + " - "+ err);
+            res.sendStatus(500);
+        }else{
+            res.sendStatus(contacts);
```

Comment:
Please, change this code to remove the id from the info that is send to the user

Reply:
Changes done